notifications_utils/countries/__init__.py

```diff
@@ -0,0 +1,68 @@
+from functools import lru_cache
+
+from notifications_utils.columns import Columns
+from notifications_utils.sanitise_text import SanitiseASCII
+
+from .data import (
+    ADDITIONAL_SYNONYMS,
+    COUNTRIES_AND_TERRITORIES,
+    EUROPEAN_ISLANDS,
+    ROYAL_MAIL_EUROPEAN,
+    UK_ISLANDS,
+    UK_POSTAGE_REGIONS,
+    Postage,
+)
+
+
+class CountryMapping(Columns):
+
+    @staticmethod
+    @lru_cache(maxsize=2048, typed=False)
+    def make_key(original_key):
+
+        original_key = original_key.replace('&', 'and')
+        original_key = original_key.replace('+', 'and')
+
+        normalised = "".join(
+            character.lower() for character in original_key
+            if character not in " _-',.()"
+        )
+
+        if '?' in SanitiseASCII.encode(normalised):
+            return normalised
+
+        return SanitiseASCII.encode(normalised)
+
+    def __getitem__(self, key):
+        for option in (key, 'The {}'.format(key)):
```

Comment:
Do you have a test for the `The {}` case somewhere? Couldn't see one at a glance

Reply:
Added in cf6912a